src/crypto/ossl/dtls.rs

```diff
@@ -15,8 +13,9 @@ use super::io_buf::IoBuffer;
 use super::stream::TlsStream;
 use super::CryptoError;
 
-const DTLS_CIPHERS: &str = "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
-const DTLS_EC_CURVE: Nid = Nid::X9_62_PRIME256V1;
+// We restrict cipher suites to those that include ephermeral Diffie-Hellman or ephemeral
+// Elliptical Curve Diffie-Hellman AND AES-256 or AES-GCM.
+const DTLS_CIPHERS: &str = "ECDHE+AESGCM:DHE+AESGCM:ECDHE+AES256:DHE+AES256";
```

Comment:
* EECDH -> ECDHE
* EDH -> DHE

I'm trying to Google and don't quite understand this change. We already included ephemeral Diffie-Hellmans but named a bit differently. What is this change?

Reply:
Just consistency in order of the key exchange and cipher... and having the acronym orders match the typical naming I've seen. I can revert, but this seemed more consistent to me.

Reply:
Just to clarify a bit more

The Key exchange and cipher had different orders in the old string. Where it had KEY_EXCHANGE+CIPHER for 2 of the selectors, and CIPHER_KEY_EXCHANGE for the other two. So that was making the line consistent in ordering.

The EDH vs DHE and EECDH vs ECDHE is also about being consistent, but here it's about being consistent with the typical order used. DHE and ECDHE is what I've seen in openssl (eg. openssl ciphers), as well as Wireshark when it decodes packets.

This change is purely about consistency, there is no functional difference.